en/misc.ftl

```diff
@@ -267,6 +267,20 @@ first-time-user-email-hero-primary-text = As a { -brand-name-firefox } user, you
 first-time-user-email-hero-secondary-text = Manage all your masks from your { -brand-name-relay } dashboard.
 first-time-user-email-hero-cta = View your dashboard
 
+## Email sent to people onboarding that receive their first forwarded email.
+
+our-promise-header = Our promise to you
+our-promise-content = { -brand-name-relay } email masks forward to your email inbox any messages sent to that email mask—without revealing your real email address to the sender.
```

Comment:
need spaces before and after the M dash.

Reply:
Do you mean the dash in "mask-without"? The design intends for them to have no space between them

Reply:
[These are the guidelines](https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-hyphens-and-dashes-c9) from content

Reply:
If it's connected like that, it's a hyphenation, the word would be a hyphenated word would be `mask-without`. I don't think that's the meaning here.

Reply:
Is using an em-dash here ok? (its already an em dash currently)

Reply:
Nevermind, I understand why said it needs spaces now